### PR TITLE
chore: add CONTRIBUTING file

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It copies the CONTRIBUTING.md file from tc-java, locating it in the root dir of the project.

## Why is it important?
Two things: 1) it provides context on how to contribute to the project, and 2) helps with the onboarding process for Hacktoberfest

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+There are additional docs describing [contributing documentation changes](./docs/contributing_docs.md).
+
+### GitHub Sponsorship
+
+Testcontainers is [in the GitHub Sponsors program](https://github.com/sponsors/testcontainers)!
+
+This repository is supported by our sponsors, meaning that issues are eligible to have a 'bounty' attached to them by sponsors.
+
+Please see [the bounty policy page](https://golang.testcontainers.org/bounty) if you are interested, either as a sponsor or as a contributor.
```
Comment (HofmeisterAn) [inline CONTRIBUTING.md:13]: https://golang.testcontainers.org/bounty returns 404. Shouldn't we have a common page for all languages?

File: docs/bounty.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Testcontainers issue bounty policy
+
+## General
+
+We want to use issue bounties to encourage contributions in areas that are important to our sponsors, or tricky to solve.
+This includes bug fixes and new features.
+We hope that this will provide incentives to tackle issues, and gives sponsors a way to influence where development time is expended.
+We also want to reward our contributors, some of whom make huge efforts to improve Testcontainers and help their fellow developers!
+
+!!! note
```
Comment (HofmeisterAn) [inline docs/bounty.md:10]: Maybe a GitHub Markdown Emoji? Something like :exclamation: (`:exclamation:`).

File: CONTRIBUTING.md (hunk 1 above)
Comment (mdelapenya) [reply CONTRIBUTING.md:13]: It's added in this PR 😁

Comment (HofmeisterAn) [reply CONTRIBUTING.md:13]: 🤦‍♂️

**Review by eddumelendez — APPROVED** [review]

**Pull request merged**